.travis.yml

```diff
@@ -80,12 +76,17 @@ matrix:
           env: DOWNSTREAM=aws-encryption-sdk
         - python: 2.7
           env: DOWNSTREAM=dynamodb-encryption-sdk OPENSSL=1.1.0i
+          # dynamodb-encryption-sdk tests fail on xenial for whatever reason
+          dist: trust
         - python: 2.7
           env: DOWNSTREAM=certbot OPENSSL=1.1.0i
         - python: 2.7
           env: DOWNSTREAM=certbot-josepy
         - python: 2.7
           env: DOWNSTREAM=urllib3
+          # urllib3 tests fail on xenial for whatever reason
+          dist: trusty
+          sudo: false
```

Comment:
This one says sudo: false but the other trusty builds don't. I assume we should put it everywhere?

Reply:
urllib3's tests fail on `sudo: true` for whatever reason, no one else needs it